### PR TITLE
Rules and guides information in English

### LINKED ISSUE #104: English navbar directs to a Finnish page
This relates to an issue regarding having no English page (yhdistys>säännöt>index.en.md), so the fix is to create the English page and translate the content 
image

### PR DESCRIPTION
**Pull request by lurppa**
Added some information in English rules and guides page.

Closes #104

### CODE REVIEW

File: content/yhdistys/säännöt/index.en.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
++++
+
+Unfortunately we don't have the resources to translate all our rules and guides
+from Finnish, while also making sure no content gets lost in translation. Other
+than the alredy translated information in this page, the best we can provide is
+a [link to google translate](https://linkkijkl-fi.translate.goog/yhdistys/s%C3%A4%C3%A4nn%C3%B6t/?_x_tr_sl=fi&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp)
+and the best of hopes in understanding it.
+
+
+# Wesites information policy
```
Comment (ikaros02) [inline content/yhdistys/säännöt/index.en.md:14]: Wesites -> Website's/Website (?) idk

Comment (lurppa) [reply content/yhdistys/säännöt/index.en.md]: Went with "Website"

File: content/yhdistys/säännöt/index.en.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
++++
+title = "Rules and Guides"
+description = "Rules and guides"
+keywords = ["rules", "guides"]
++++
+
+Unfortunately we don't have the resources to translate all our rules and guides
+from Finnish, while also making sure no content gets lost in translation. Other
+than the alredy translated information in this page, the best we can provide is
```
Comment (45spoons) [inline content/yhdistys/säännöt/index.en.md:9]: typo "alredy"

File: content/yhdistys/säännöt/index.en.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+a [link to google translate](https://linkkijkl-fi.translate.goog/yhdistys/s%C3%A4%C3%A4nn%C3%B6t/?_x_tr_sl=fi&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp)
+and the best of hopes in understanding it.
+
+
+# Wesite Information Policy
```
Comment (45spoons) [inline content/yhdistys/säännöt/index.en.md:14]: typo "wesite"

**Review by 45spoons — APPROVED** [review]
Brilliant, great getting changes made even at this time of year, thanks!

**Pull request merged**